src/Session.hs

```diff
@@ -125,6 +134,40 @@ sessionRestart session@Session{..} = do
     sessionStart session cmd setup
 
 
+performEvals :: Ghci -> Bool -> [FilePath] -> IO [Load]
```

Comment:
I'm wondering if Load should contain PerformEvals or not? Should it return a separate type, and then higher level things work with Either Load EvalResult? The Session seems to mostly deal with evals (as seems entirely reasonable), but the Load data type is defined in the ghcid library, which means the data type appears there despite not being used there.

Reply:
I don't have enough context of how ghcid works to have an opinion here. I didn't even realize there was a library vs an app, but it would explain why `stack repl` refuses to load the project! Happy to do whatever you suggest, though this seemed like the most straightforward impl.